tests/spec/puppeteer-fs-spec.js

```diff
@@ -13,13 +13,26 @@ describe("Puppeteer filesystem tests for the Python Editor.", function() {
     beforeAll(async () => {
         // Setup a headless Chromium browser.
         // Flags allow Puppeteer to run within a container.
-        browser = await puppeteer.launch({headless: true, args: ["--no-sandbox", "--disable-setuid-sandbox", "--disable-dev-shm-usage"]});
+        browser = await puppeteer.launch({headless: false, args: ["--no-sandbox", "--disable-setuid-sandbox", "--disable-dev-shm-usage"]});
     });
 
     afterAll(async () => {
         browser.close();
     });
 
+    it("Correctly handles a file with an invalid extension", async function(){
```

Comment:
This tests is not specific to the filesystem, so maybe we should move it to the generic pupetter-spec file.